.github/workflows/ci-test.yaml

```diff
@@ -13,3 +13,19 @@ jobs:
           VERBOSE: true
         run: |
           make local-test
+  stress-test-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: make stress-test
+        env:
+          VERBOSE: true
+        run: |
+          make stress
+  integration-test-on-kind:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: run tests
+        run: |
+          SONOBUOY_CLI=../../build/linux/amd64/sonobuoy ./scripts/run_integration_tests.sh
```

Comment:
What's going on here? Is the sonobuoy binary taking script as param ? Or should the script generate something that sonobuoy can consume as param?

Reply:
that's the path to the Sonobuoy binary, so that the script can find where the newly built binary is (since it's not in $PATH)

https://github.com/vmware-tanzu/sonobuoy/blob/master/Makefile#L158-L161

Reply:
LoL.  Oh jee. Yesterday was distracting.  Oh man.